tests/model_fields/test_charfield.py

```diff
@@ -32,6 +33,14 @@ def test_emoji(self):
         p.refresh_from_db()
         self.assertEqual(p.title, 'Smile 😀')
 
+    def test_add(self):
```

Comment:
This probably isn't the best place for the test -- it's testing changes in expressions, not in model fields. Try `tests/expressions` instead.